src/steamship/app/lambda_handler.py

```diff
@@ -83,7 +91,57 @@ def _handler(event: Dict, _: Dict = None) -> Response:
             )
 
     def handler(event: Dict, context: Dict = None) -> dict:
-        response = _handler(event, context)
+        logger = None
+        loggingConfig = event.get("loggingConfig")
+
+        if loggingConfig is None:
+            return Response.error(
+                code=HTTPStatus.INTERNAL_SERVER_ERROR,
+                message="Plugin/App handler did not receive a remote logging config.",
+            ).dict(by_alias=True)
+
+        loggingHost = loggingConfig.get("loggingHost")
+        loggingPort = loggingConfig.get("loggingPort")
+
+        logger = logging.getLogger("lambda.handler")
+        logging.basicConfig(level=logging.INFO)
+        loggingHandler = None
+        # This log statement intentionally goes to the DEFAULT logging handler, to debug logging configuration issues
+        logging.info(f"Logging host: {loggingHost} Logging port: {loggingPort}")
+
+        if (
+            loggingHost != "none"
+        ):  # Key off the string none, not 'is None', to avoid config errors where remote host isn't passed
+            # Configure remote logging
+            if loggingHost is None:
```

Comment:
I'd move these up higher with the other check for code readability

Reply:
You need the stuff in between to have happened though?